distribution/debian/scripts/postinst

```diff
@@ -1,8 +1,13 @@
 #!/bin/sh
 set -e
 
-usermod -aG systemd-journal vector || true
+# Add Vector to adm group to read /var/logs
+usermod --append --gid adm vector || true
```

Comment:
long flags :heart:

Reply:
Heh. I guess we can do that from now one... :)